src/Microsoft.AspNet.Mvc.ViewFeatures/ViewFeatures/IHtmlGenerator.cs

```diff
@@ -22,6 +22,7 @@ public interface IHtmlGenerator
         string FormatValue(object value, string format);
 
         TagBuilder GenerateActionLink(
```

Comment:
Time to invoke the "you touched it; you document it" rule? Perhaps but at most for the two methods you've touched.